modules/installation-overview.adoc

```diff
@@ -0,0 +1,130 @@
+// Module included in the following assemblies:
+//
+// * installation/installing-quickly-cloud.adoc
+// * installation/installing-customizations-cloud.adoc
+// * installation/installing-existing-hosts.adoc
+
+[id='installation-overview-{context}']
+= {product-title} installation overview
+
+You use the {product-title} installer to create {product-title} clusters in
+a variety of environments. The same installer allows you to install
+{product-title} on your own infrastructure or in Amazon Web Services (AWS).
+By default, the installer acts as an installation wizard, prompting you
+for values that it cannot determine on its own and providing reasonable default
+values for the remaining parameters. You can also customize the installation
+process to support advanced infrastructure scenarios.
+
+In supported cloud environments, the installer can also provision the underlying
+infrastructure for the cluster. If possible, use this feature to avoid
+having to provision and maintain the cluster infrastructure. In all other 
+environments, you use the installer to generate the assets that you need to
+provision your cluster infrastructure.
+
+[NOTE]
+====
+In {product-title} version 4.0, the installer can only provision infrastructure
+for clusters on AWS. If you use another cloud provider or on-premises hardware, 
+you must provide the infrastructure. 
+====
+
+[discrete]
+== Cluster installation process
+
+In supported cloud environment scenarios, {product-title} manages all aspects of
+the cluster, including the operating system itself. Each machine boots with a
+configuration that references resources hosted in the cluster that it joins.
+This configuration allows the cluster to manage itself as updates are applied.
+
+Because each machine in the cluster requires information about the cluster when
```

Comment:
I don't think this is as clear about _why_ the bootstrap machine is necessary. There is a line in the installer docs which helps explain this:

> A downside to this approach, however, is that new clusters have no way of starting without external help - every machine in the to-be-created cluster is waiting on the to-be-created cluster.